### DISCUSSION TITLE
Collision box aligned with model

### DISCUSSION THREAD

**Discussion by oh-wind** [dimforge/rapier#640] [Q&A]
When setting the height of the collider to be larger than the model, the model gets moved to the center. How can I keep the model at the bottom? I'm using bevy_rapier3d

![Image](https://github.com/dimforge/rapier/assets/34809611/35b6526b-e1a1-4c45-aea5-6b600c7a7bcc)

**Comment by ThierryBerger** [comment]
rapier doesn't really handle the model positionning, so I imagine you might want to place your collider higher, or your model lower ? Do you have some example code ?
